model/src/schema/schema.ts

```diff
@@ -169,6 +169,14 @@ const feedbackSchema = joi.object().keys({
     is: joi.boolean().valid(false),
     then: joi.string().optional(),
   }),
+  email: joi
```

Comment:
I'll never win .. an email is the thing that's sent to an email address :)

Reply:
That's my bad :p.. changing. Who says I never let you win 🏆